source/indexes/multikey-index.txt

```diff
@@ -0,0 +1,79 @@
+.. _c-multikey-index:
+
+================
+Multikey Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Multikey indexes** are indexes that improve performance for queries that specify a field
+that contains an array value. You can define a multikey index by using the same syntax as
```

Comment:
S: since this sentence repeats "that" a few times, I think you can shorten it:
```suggestion
**Multikey indexes** are indexes that improve performance for queries on array-valued fields. You can define a multikey index by using the same syntax as
```